xcp_d/utils/hcp2fmriprep.py

```diff
@@ -15,154 +15,226 @@
-    whiteL = glob.glob(anat_dirx + "/fsaverage_LR32k/*L.white.32k_fs_LR.surf.gii")[0]
-
-    hcpfiles = [
-        tw1,
-        segm,
-        ribbon,
-        brainmask,
-        tw1,
-        tw1,
-        midL,
-        midR,
-        pialL,
-        pialR,
-        whiteL,
-        whiteR,
-        infL,
-        infR,
-    ]
+    t1w_orig = os.path.join(in_dir, "T1w_restore.nii.gz")
+    brainmask_orig = os.path.join(in_dir, "brainmask_fs.nii.gz")
+    ribbon_orig = os.path.join(in_dir, "ribbon.nii.gz")
+    dseg_orig = os.path.join(in_dir, "aparc+aseg.nii.gz")
+
+    fsaverage_dir_orig = os.path.join(in_dir, "fsaverage_LR32k")
+
+    # NOTE: Why glob? Do we not know the full filenames? Are there multiple files?
+    rh_midthickness_orig = glob.glob(
+        os.path.join(fsaverage_dir_orig, "*R.midthickness.32k_fs_LR.surf.gii")
+    )[0]
+    lh_midthickness_orig = glob.glob(
+        os.path.join(fsaverage_dir_orig, "*L.midthickness.32k_fs_LR.surf.gii")
+    )[0]
+    rh_inflated_orig = glob.glob(
+        os.path.join(fsaverage_dir_orig, "*R.inflated.32k_fs_LR.surf.gii")
+    )[0]
+    lh_inflated_orig = glob.glob(
+        os.path.join(fsaverage_dir_orig, "*L.inflated.32k_fs_LR.surf.gii")
+    )[0]
+
+    rh_pial_orig = glob.glob(os.path.join(fsaverage_dir_orig, "*R.pial.32k_fs_LR.surf.gii"))[0]
+    lh_pial_orig = glob.glob(os.path.join(fsaverage_dir_orig, "*L.pial.32k_fs_LR.surf.gii"))[0]
+
+    rh_wm_orig = glob.glob(os.path.join(fsaverage_dir_orig, "*R.white.32k_fs_LR.surf.gii"))[0]
+    lh_wm_orig = glob.glob(os.path.join(fsaverage_dir_orig, "*L.white.32k_fs_LR.surf.gii"))[0]
 
     # to fmriprep directory
-    if "sub" not in sub_id:
-        subid = "sub-" + sub_id
-    t1wim = anatdir + subid + "_desc-preproc_T1w.nii.gz"
-    t1seg = anatdir + subid + "_dseg.nii.gz"
-    t1ribbon = anatdir + subid + "_desc-ribbon_T1w.nii.gz"
-    t1brainm = anatdir + subid + "_desc-brain_mask.nii.gz"
-    regfile1 = anatdir + subid + "_from-T1w_to-MNI152NLin2009cAsym_mode-image_xfm.txt"
-    regfile2 = anatdir + subid + "_from-MNI152NLin2009cAsym_to-T1w_mode-image_xfm.txt"
-
-    lMid = anatdir + subid + "_hemi-L_midthickness.surf.gii"
-    rMid = anatdir + subid + "_hemi-R_midthickness.surf.gii"
-
-    lpial = anatdir + subid + "_hemi-L_pial.surf.gii"
-    rpial = anatdir + subid + "_hemi-R_pial.surf.gii"
-
-    lwhite = anatdir + subid + "_hemi-L_smoothwm.surf.gii"
-    rwhite = anatdir + subid + "_hemi-R_smoothwm.surf.gii"
-
-    linf = anatdir + subid + "_hemi-L_inflated.surf.gii"
-    rinf = anatdir + subid + "_hemi-R_inflated.surf.gii"
-    newanatfiles = [
-        t1wim,
-        t1seg,
-        t1ribbon,
-        t1brainm,
-        regfile1,
-        regfile2,
-        lMid,
-        rMid,
-        lpial,
-        rpial,
-        lwhite,
-        rwhite,
-        linf,
-        rinf,
-    ]
+    t1w_fmriprep = os.path.join(anat_dir_fmriprep, f"{sub_id}_desc-preproc_T1w.nii.gz")
```

Comment:
If this is in standard space, then we should include the space label in the filename.

Reply:
Thanks to @perronea and @mattcieslak, it looks like they're in MNI152NLin6Asym.